modules/docs/arrow-docs/docs/docs/quickstart/libraries/README.md

```diff
@@ -18,156 +19,264 @@ In this doc we'll describe all the modules that form the core, alongside a list
 {:.beginner}
 beginner
 
-The smallest set of [datatypes]({{ '/docs/datatypes/intro/' | relative_url }}) necessary to start in FP, and that other libraries can build upon.
+```groovy
+dependencies {
+    compile "io.arrow-kt:arrow-core:$arrow_version"
+}
+```
+
+This library include the possible [typeclass extensions]({{ '/docs/patterns/glossary/#instances-and-extensions-interfaces' | relative_url }}) including datatypes and typeclasses from `arrow-core-data` that can be implemented for the datatypes in their respective libraries.
```

Comment:
No need to mention any of this here, README should just include a way to get the main artifact loaded in a project. This can be explained in the advanced page

Reply:
this is the advanced page.